coriolis/api/v1/views/migration_view.py

```diff
@@ -27,10 +27,14 @@ def transform(key, value):
     migration_dict = dict(itertools.chain.from_iterable(
         transform(k, v) for k, v in migration.items()))
 
-    execution = replica_tasks_execution_view.format_replica_tasks_execution(
-        req, migration_dict["executions"][0])
+    if len(migration_dict["executions"]):
```

Comment:
NIT: I'd make this a `.get("execution", [])` to be extra safe, though afaik every Migration automatically has an execution.